modules/cli/src/main/java/org/apache/ignite/internal/cli/call/cluster/status/ClusterState.java

```diff
@@ -38,15 +39,26 @@ public class ClusterStatus {
 
     private final List<String> metadataStorageNodes;
 
-    private ClusterStatus(int nodeCount, boolean initialized, String name,
-            boolean connected, String nodeUrl, List<String> cmgNodes, List<String> metadataStorageNodes) {
+    private final ClusterStatus clusterStatus;
+
+    private ClusterState(
+            int nodeCount,
+            boolean initialized,
+            String name,
+            boolean connected,
+            String nodeUrl,
+            List<String> cmgNodes,
+            List<String> metadataStorageNodes,
+            ClusterStatus clusterStatus
+    ) {
         this.nodeCount = nodeCount;
         this.initialized = initialized;
         this.name = name;
         this.connected = connected;
         this.nodeUrl = nodeUrl;
         this.cmgNodes = cmgNodes;
         this.metadataStorageNodes = metadataStorageNodes;
```

Comment:
How about making defensive copies? Better safe than sorry, and this class should not have a huge amount of instances